pkg/cluster/tekton/collector/collector.go

```diff
@@ -44,6 +46,13 @@ type (
 	}
 )
 
+type Log struct {
```

Comment:
Why use a struct like this?

Reply:
Looks like another Log has been defined in core/controller/pipelinerun/controller.go